src/ceph.in

```diff
@@ -225,7 +225,7 @@ def validate_target(target):
                   file=sys.stderr)
             return False
 
-        if service_id in exist_ids:
+        if service_id in exist_ids or len(exist_ids) > 0 and service_id == '*':
```

Comment:
i am reading http://tracker.ceph.com/issues/21191, but http://docs.ceph.com/docs/master/rados/configuration/ceph-conf/#runtime-changes does not document the `ceph` cli that it supports following command:
```sh
$ bin/ceph tell mds.\* help
```

Reply:
should we support this command?

Reply:
i don't really think so. @batrick what do you think?

Reply:
No, I don't think we need to either.

Reply:
I'm going to go ahead and merge this just because it's inconsistent with the docs and the code may be reused in the future for something else and not behave properly.